TypeScript.YAML-tmLanguage

```diff
@@ -339,7 +339,7 @@ repository:
     name: meta.type.function.return.ts
     begin: '=>'
     beginCaptures:
-      '0': { name: keyword.others.ts }
+      '0': { name: keyword.operator.ts }
```

Comment:
@aeschli suggested to make to [`keyword.operand`](https://github.com/Microsoft/TypeScript-TmLanguage/pull/105#issuecomment-195747412), does it make a difference? Or maybe he meant `operator`